compose/aspnet-mssql-compose.md

```diff
@@ -0,0 +1,187 @@
+---
+description: Create a Docker Compose application using ASP.NET Core and SQL Server on Linux in Docker. 
+keywords: dotnet, .NET, Core, example, ASP.NET Core, SQL Server, mssql
+title: "Quickstart: Compose and ASP.NET Core with SQL Server"
+---
+
+This quick-start guide demonstrates how to use Docker Engine on Linux and Docker
+Compose to set up and run the sample ASP.NET Core application using the
+[ASP.NET Core Build image](https://hub.docker.com/r/microsoft/aspnetcore-build/)
+with the
+[SQL Server on Linux image](https://hub.docker.com/r/microsoft/mssql-server-linux/).
+You just need to have [Docker Engine](https://docs.docker.com/engine/installation/)
+and [Docker Compose](https://docs.docker.com/compose/install/) installed on your
+platform of choice: Linux, Mac or Windows.
+
+For this sample, we will create a sample `dotnet core web application` using the
+`aspnetcore-build` Docker image. After that, we will create a `Dockerfile`,
+configure this app to use our SQL Server database, and then create a
+`docker-compose.yml` that will define the behavior of all of these components.
+
+> **Note**: This sample is made for Docker Engine on Linux. For Windows
+> Containers, visit
+> [Docker Labs for Windows Containers](https://github.com/docker/labs/tree/master/windows).
+
+1.  Create a new directory for your application.
+
+    This directory will be the context of your docker-compose project. For
+    [Docker for Windows](https://docs.docker.com/docker-for-windows/#/shared-drives) and
+    [Docker for Mac](https://docs.docker.com/docker-for-mac/#/file-sharing), you
+    have to set up file sharing for the volume that you need to map. 
+
+1.  Within your directory, use the `aspnetcore-build` Docker image to generate a
+    sample web application within the container under the `/app` directory and
+    into your host machine in the working directory:
+
+    ```bash
+    $ docker run -v ${PWD}:/app --workdir /app microsoft/aspnetcore-build:lts dotnet new mvc --auth Individual
+    ```
+
+    > **Note**: If running in Docker for Windows, make sure to add the necessary
+    > `C:/` prefix to the host mapped folder and that you are running on Linux
```

Comment:
The note was applicable if they were specifying the path themselves. But now that you mention it, I noticed that this command will only work on a Powershell terminal. Do you guys work with CMD at all in your other guides?